src/backend/marsha/core/api/live_registration.py

```diff
@@ -280,10 +280,7 @@ def set_display_name(self, request):
                     defaults=update_fields,
                 )
             return Response(
```

Comment:
Suggestion fix commit message:

When the update display_name endpoint is fetch, we want to return the
full serizlied liveregistration object tu use it then in the front
application.
=>
When the update display_name endpoint **is fetched**, we want to return the
full **serialised** liveregistration object **to** use it then in the front
application.